### DISCUSSION TITLE
What file type should I use for JavaScript and Processing JavaScript?

### DISCUSSION THREAD

**Discussion by opticaldrive** [community/community#61533] [Programming Help] [answered]
### Body

What file type should I use for JavaScript and Processing JavaScript?
Can't recall.
Thank you.

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by GhostOf0days** [comment, marked as answer]
.js

**Reply by GhostOf0days** [reply]
For instance, example.js.

**Reply by opticaldrive** [reply]
Thank you!
It is also .js for PJS right?
Thanks!

**Reply by GhostOf0days** [reply]
@V205Arduino Yes. However, it is deprecated and no longer maintained or supported. I reccomend checking out p5.js instead.

**Reply by opticaldrive** [reply]
Will do! 
Thank you!
